UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.cs

```diff
@@ -86,7 +87,7 @@ void UpdateMe()
 	/// </summary>
 	/// <param name="sceneName"></param>
 	/// <returns></returns>
-	IEnumerator LoadSubScene(string sceneName, SubsceneLoadTimer loadTimer = null, bool HandlSynchronising = true,
+	public IEnumerator LoadSubScene(string sceneName, SubsceneLoadTimer loadTimer = null, bool HandlSynchronising = true,
```

Comment:
where are you calling this from?

Reply:
![image](https://github.com/user-attachments/assets/e3ddfb50-57f7-4597-a405-c6b0879de766)